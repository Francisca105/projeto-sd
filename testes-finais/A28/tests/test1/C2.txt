read <a,[0-9]>
sleep 2
take <a,2>
exit
