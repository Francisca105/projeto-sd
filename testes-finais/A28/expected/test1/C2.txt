ClientMain
> OK
<a, 2>

> > OK
<a, 1>

> 